subsys/fs/fuse_fs_access.c

```diff
@@ -496,17 +496,19 @@ static void fuse_fs_access_init(void)
 static void fuse_fs_access_exit(void)
 {
 	char *full_cmd;
+	int full_cmd_length;
 	const char cmd[] = "fusermount -uz ";
 
 	if (fuse_mountpoint == NULL) {
 		return;
 	}
 
-	full_cmd = malloc(strlen(cmd) + strlen(fuse_mountpoint) + 1);
+	full_cmd_length = strlen(cmd) + strlen(fuse_mountpoint) + 1;
+	full_cmd = malloc(full_cmd_length);
 
-	sprintf(full_cmd, "%s%s", cmd, fuse_mountpoint);
+	snprintk(full_cmd, full_cmd_length, "%s%s", cmd, fuse_mountpoint);
```

Comment:
Note that this function is called outside of the zephyr kernel context so no zephyr specific calls  should be made

Reply:
Removed